assessments/README.md

```diff
@@ -15,16 +15,15 @@ The primary goal is to reduce the risk from malicious attacks and accidental bre
 
 ### 2) Accelerate adoption of cloud native technologies
 
-Security reviews are a necessary, yet time consuming process, where each
+Security reviews are a necessary, time intensive process. Each
 company, organization and project must perform its own reviews to ensure
 that it meets its unique commitments to its own users and stakeholders.
-In open source, simply finding security-related information can be a very
-time consuming part of the the process. The process is designed to enable improved discovery of security information & streamlined security reviews in multiple ways:
+In open source, simply finding security-related information can be overwhelmingly difficult and a time consuming part of the security review. The CNCF security assessment process is intended to enable improved discovery of security information & assist in streamlining internal and external security reviews in multiple ways:
 
```

Comment:
I realize the original text didn't conform to line wrapping on rest of the doc... could you please add some line breaks?

Reply:
I've gone through this read me and removed the strange breaks and did some consistent formatting, hopefully it'll look btr.